models/external-accounts.js

```diff
@@ -11,4 +11,33 @@ const addExternalAccountData = async (data) => {
   }
 };
 
-module.exports = { addExternalAccountData };
+const fetchExternalAccountData = async (query, param) => {
+  try {
+    const userExternalAccountData = [];
+
+    let externalAccountData;
+    externalAccountData = await externalAccountsModel.where("token", "==", param);
+
+    Object.keys(query).forEach((key) => {
```

Comment:
It can be done without forEach this way
```js
if (query && query?.['type']) {
  externalAccountData = externalAccountData.where("type", "==", query?.['type']);
}
```

Reply:
Done with change.